File: python/sprint1_nonfinals/d.py
from typing import List

def get_weather_randomness(temperatures: List[int]) -> int:
    # Здесь реализация вашего решения
    pass

def read_input() -> List[int]:
    n = int(input())
    temperatures = list(map(int, input().strip().split()))
    return temperatures

temperatures = read_input()
print(get_weather_randomness(temperatures))

File: cpp/sprint1_nonfinals/d.cpp
#include <iostream>
#include <vector>

using namespace std;

int GetWeatherRandomness(const vector <int> &temperatures) {
	// Здесь реализация вашего решения

}

void print(int result) {
	cout << result << endl;
}

int main() {
	int n;
	cin >> n;
	vector <int> temperatures(n);
	for (int i = 0; i < n; ++i)
		cin >> temperatures[i];
	print(GetWeatherRandomness(temperatures));
}
